tests/integration/standard/test_types.py

```diff
@@ -704,6 +704,7 @@ def test_can_insert_tuples_with_nulls(self):
         self.assertEqual(('', None, None, b''), result[0].t)
         self.assertEqual(('', None, None, b''), s.execute(read)[0].t)
 
+    @scylla_only
```

Comment:
Why only on scylla ?

Reply:
this test was introduce as part of fixing this issue:
https://github.com/scylladb/python-driver/issues/201

the behavior of scylla and cassandra isn't the same for what covered in this test

@Lorak-mmk might remember the details, as he introduced this test